packages/typescriptlang-org/src/copy/inYourLanguage.ts

```diff
@@ -17,7 +17,7 @@ export const inYourLanguage: Record<
   pt: {
     shorthand: "Em Pt",
     body: "Esta página está disponível em Português",
-    open: "Ir",
+    open: "Trocar Língua",
```

Comment:
```suggestion
    open: "Trocar língua",
```
![image](https://user-images.githubusercontent.com/17657014/88690298-e364d580-d0d1-11ea-884f-3beb2d418d9a.png)
![image](https://user-images.githubusercontent.com/17657014/88690325-eb247a00-d0d1-11ea-9e21-36a76aa40e02.png)

Looks better IMO